deps/wazuh_testing/wazuh_testing/data_visualization_module/dash_app/app.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Data Visualization Script.
+
+This script executes the data visualization module using a Dash application. In addition, it contains some functions
+to process the YAML configuration file that is passed as an argument to the script. This file must contain the type
+of component to be displayed (agent, manager, dashboard or indexer), the processes to be displayed depending on the
+component ( for example, daemons) and the columns of the CSV files that are not to be displayed (for example, commit
+or version).
+"""
+
+import os
+from typing import Any, Dict
+
+import argparse
+import yaml
+
+from dash import Dash
+
+from cache import cache
+from callbacks import callbacks
+from layout import create_layout
+
+
+def load_config(config_file: str) -> Dict[str, Any]:
+    """Function for loading the file containing the configuration items for visualization in YAML format.
+
+    Args:
+        config_file (str): file containing configuration items.
+
+    Returns:
+        config (Dict[str, Any]): dictionary containing information from the YAML file.
+    """
+    with open(config_file) as file:
+        config = yaml.safe_load(file)
+    return config
+
+
+def get_arguments() -> argparse:
```

Comment:
Use Namespace from argparse

```suggestion
def get_arguments() -> Namespace:
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5716/commits/309f6d462764a633386973c905c0e7a0d5c0ce88